checks/python_module_version.go

```diff
@@ -0,0 +1,68 @@
+package checks
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func init() {
+	availableChecks["python-module-version"] = func(args map[string]interface{}) (Checker, error) {
+		return PythonModuleVersion{}.FromArgs(args)
+	}
+}
+
+// PythonModuleVersion checks if python module is installed on the system
+// And verifies its version.
+//
+// Type:
+//   - python-module-version
+//
+// Support Platforms:
+//   - Linux
+//   - Windows
+//
+// Argument:
+//   name (required): A string value that represents the gem name
+//   version (optional): An optional version number to check
+// Leave version blank, if you just want to verify that a module is present
+type PythonModuleVersion struct {
+	Name    string
+	Version string
+}
+
+// Check if a python module is installed on the system and verify version if
+// the Version string is set
+func (pmv PythonModuleVersion) Check() error {
+	pyCommand := "import " + pmv.Name + "; print(" + pmv.Name + ".__version__)"
+	out, err := exec.Command("python", "-c", pyCommand).Output()
+	if err != nil {
+		return fmt.Errorf("%s isn't installed and should be: %s", pmv.Name, err)
+	}
+
+	if len(pmv.Version) >= 1 {
+		if string(pmv.Version) != strings.TrimRight(string(out), "\n") {
+			return fmt.Errorf("%s has version %s and it should be version: %s", pmv.Name, out, pmv.Version)
+		}
+	}
+
+	return nil
+}
+
+// FromArgs will populate the PythonModuleVersion struct with the args given in the tests YAML
+// config
+func (pmv PythonModuleVersion) FromArgs(args map[string]interface{}) (Checker, error) {
+	if err := requiredArgs(args, "name"); err != nil {
+		return nil, err
+	}
+
+	if err := decodeFromArgs(args, &pmv); err != nil {
+		return nil, err
+	}
+
+	if _, versionGiven := args["version"]; pmv.Version == "" && !versionGiven {
```

Comment:
The zero value for a string in go is "" you don't need this if statement here since if it's not provided the Version will already be ""